packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -44,15 +45,15 @@ function BlockListAppender( {
 			<Inserter
 				rootClientId={ rootClientId }
 				renderToggle={ ( { onToggle, disabled, isOpen } ) => (
-					<IconButton
-						label={ __( 'Add block' ) }
-						icon="insert"
-						onClick={ onToggle }
+					<Button
```

Comment:
As an aside, an alternative would be adding an option like `isLayoutVertical` to IconButton that switches the flex direction. 

I'm also ok with this solution and think in interests of merging this earlier lets keep this as is, but if we start to notice this icon button style emerge it might be nice to formalise it.

Reply:
> As an aside, an alternative would be adding an option like `isLayoutVertical` to IconButton that switches the flex direction.
> 
> I'm also ok with this solution and think in interests of merging this earlier lets keep this as is, but if we start to notice this icon button style emerge it might be nice to formalise it.

I agree with all of the above (including not letting this be a blocker), in that it strikes me as odd to have an `IconButton` offering and choose instead to render `<Button><Icon /> Text</Button>` instead, since it implies some lack of flexibility in the former.

Granting at the same time, there's some [prior discussion](https://github.com/WordPress/gutenberg/issues/7534#issuecomment-461429534) about whether `IconButton` really _should_ exist as a separate thing.

Reply:
Agreed. I don't think we should allow this to become a Blocker.